modules/virt-creating-data-volumes-using-pvc-api.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-pvc-api_{context}"]
+
+= Creating data volumes using the PVC API
+
+When you create a data volume using the *pvc* API, the CDI creates the data volume based on your explicit:
+
+* `accessModes` (`ReadWriteOnce`, `ReadWrtieMany`, `ReadOnlyMany`)
+* `volumeMode` (`Filesystem` or `Block`)
+* `capacity` of `storage` (`5Gi`, for example)
+
+When you explicitly allocate a data volume by using the *pvc* API, you must consider file system overhead. File system overhead is the amount of space required by the CDI to manage the file. Failing to estimate file system overhead results in a storage allocation that is less than you requested.
+
+If you use the *storage* API, the space required for file system overhead is automatically calculated for you, and the amount of space you allocate is exactly what you request.
+
+In the following YAML, using the *pvc* API allocates a data volume with a storage capacity of two gigabytes. In this case, you specify an access mode of `ReadWriteMany` to ensure that many nodes can mount the volume for reading and writing. You specify `Block` storage, using the `volumeMode` attribute, as opposed to `Filesystem`.
```

Comment:
Perhaps mention that you must know the values that your storage system can support.

Reply:
How's this:
_Because you know the values your system can support, you specify `Block` storage, using the `volumeMode` attribute, as opposed to `Filesystem`._

I was looking for a way to frame choose block over filesystem, so that's a cool, generic way to say it.....